Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "retry-policies"
-version = "0.5.0"
+version = "0.5.1"
```

❓ Are we considering this a bug fix or a general improvement? I would worry that this could catch people out as the retry policy implementation changes with a minor version bump. 

 ```suggestion
version = "0.6.0"
```

---

I doubt that anyone is relying on this specific aspect of the jittering.

I could be wrong, but I'd be very surprised!